aviary.yaml

```diff
@@ -0,0 +1,22 @@
+version: 1
+
+exclude:
+  - tests?/
```

Comment:
@infosec-wf Could we also exclude all `.md` files? We mention a few things like `x-xsrf-token` that get picked up by raven.